tools/vms-generator/utils/utils.go

```diff
@@ -99,6 +100,24 @@ const defaultInterfaceName = "default"
 const enableNetworkInterfaceMultiqueueForTemplate = true
 const EthernetAdaptorModelToEnableMultiqueue = "virtio"
 
+const (
+	cloudConfigHeader = "#cloud-config"
+
+	cloudConfigInstallAndStartService = `packages:
+  - nginx
+runcmd:
+  - [ "systemctl", "enable", "--now", "nginx" ]`
```

Comment:
I think you can convert it to string instead https://cloudinit.readthedocs.io/en/latest/topics/modules.html?highlight=runcmd#runcmd
Also it may require sudo, we provision the tests image with cloud init as well and we use sudo for systemd calls
https://github.com/kubevirt/kubevirtci/blob/main/cluster-provision/images/vm-image-builder/fedora-with-test-tooling/cloud-config#L37

long story short:
`sudo systemctl enable --now nginx`